src/util/tutorial.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import type { EdgeAccount } from 'edge-core-js'
+
+const TUTORIAL = 'tutorial.json'
+
+export const getUserLoginCount = async (account: EdgeAccount): Promise<number> => {
+  const { disklet, id } = account
```

Comment:
do not use the account disklet.

this setting should be per device and local only

Reply:
fixed